actions.go

```diff
@@ -25,6 +25,11 @@ import (
 	"strings"
 )
 
+var (
+	// osExit allows `os.Exit()` to be stubbed during testing.
+	osExit = os.Exit
```

Comment:
This change is "necessary" to get test coverage for the `os.Exit()` call in `Fatalf()`.

However, the "uncertainty" / non-`const`-ness of a `var` here may not be worth it to you, so I am happy to revert this change.

Going in the **OTHER** direction, we could also put an `ExitFunc` on `Action` the same way we did for `GetenvFunc`. (This would also help end users with testing. Right now, I've kept `osExit` internal so end users won't be able to swap it out in a test the way I have here.) If we do decide to go this route I think it'd probably be better to do it in a different PR (and I can walk back the `osExit()` change in this PR).

Reply:
I generally prefer this approach like you have it.